reconcile/terraform_tgw_attachments.py

```diff
@@ -114,8 +121,15 @@ def run(dry_run, print_only=False,
     settings = queries.get_app_interface_settings()
     clusters = [c for c in queries.get_clusters()
                 if c.get('peering') is not None]
-    ocm_map = OCMMap(clusters=clusters, integration=QONTRACT_INTEGRATION,
-                     settings=settings)
+    with_ocm = any([c.get('ocm') for c in clusters])
```

Comment:
Nit: you don't really need to build the intermediate list:
```suggestion
    with_ocm = any(c.get('ocm') for c in clusters)
```

Reply:
Interesting! what is the intermediate data type? a set?

Reply:
Hmm I guess it just calls `__iter__`